.github/workflows/test-aks.yaml

```diff
@@ -0,0 +1,369 @@
+            path: ./samples/eshop/eshop.bicep
+            uiTestFile: tests/eshop/eshop.app.spec.ts
+            deployArgs: -p environment='/planes/radius/local/resourceGroups/eshop-containers/providers/Applications.Core/environments/containers'
+          - name: eshop-azure
+            os: ubuntu-latest-m
+            runOnPullRequest: true
+            app: eshop
+            env: azure
+            path: ./samples/eshop/eshop.bicep
+            uiTestFile: tests/eshop/eshop.app.spec.ts
+            deployArgs: -p environment='/planes/radius/local/resourceGroups/eshop-azure/providers/Applications.Core/environments/azure'
+            credential: azure
+          - name: eshop-aws
+            os: ubuntu-latest-m
+            runOnPullRequest: true
+            app: eshop
+            env: aws
+            path: ./samples/eshop/eshop.bicep
+            uiTestFile: tests/eshop/eshop.app.spec.ts
+            deployArgs: -p environment='/planes/radius/local/resourceGroups/eshop-aws/providers/Applications.Core/environments/aws'
+            credential: aws
+    env:
+      BRANCH: ${{ github.base_ref || github.ref_name }}
+      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      AZURE_LOCATION: eastus
+      AKS_RESOURCE_GROUP: samples-test-rg
+      AKS_CLUSTER_NAME: samples-test-aks
+      AWS_REGION: us-west-2
+      AWS_ZONES: us-west-2a,us-west-2b,us-west-2c
+    steps:
+      # Setup the test assets and configuration
+      - name: Generate output variables
+        id: gen-id
+        run: |
+          RUN_IDENTIFIER=${{ env.RUN_IDENTIFIER }}-${{ matrix.name }}
+
+          if [[ "${{ github.event_name }}" == "pull_request" && "${{ matrix.runOnPullRequest }}" == "false" ]]; then
+            RUN_TEST=false
+          else
+            RUN_TEST=true
+          fi
+
+          # Set output variables to be used in the other jobs
+          echo "RUN_IDENTIFIER=${RUN_IDENTIFIER}" >> $GITHUB_OUTPUT
+          echo "TEST_AZURE_RESOURCE_GROUP=rg-${RUN_IDENTIFIER}" >> $GITHUB_OUTPUT
+          echo "TEST_EKS_CLUSTER_NAME=eks-${RUN_IDENTIFIER}" >> $GITHUB_OUTPUT
+          echo "RUN_TEST=${RUN_TEST}" >> $GITHUB_OUTPUT
+      - name: Checkout code
+        if: steps.gen-id.outputs.RUN_TEST == 'true'
+        uses: actions/checkout@v3
+      - name: Ensure inputs.version is valid semver
+        if: steps.gen-id.outputs.RUN_TEST == 'true' && inputs.version != ''
+        run: |
+          python ./.github/scripts/validate_semver.py ${{ inputs.version }}
+      - name: Setup Node
+        if: steps.gen-id.outputs.RUN_TEST == 'true'
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - name: az CLI login
+        if: steps.gen-id.outputs.RUN_TEST == 'true'
+        run: |
+          az login --service-principal \
+            --username ${{ secrets.AZURE_SANDBOX_APP_ID }} \
+            --password ${{ secrets.AZURE_SANDBOX_PASSWORD }} \
+            --tenant ${{ secrets.AZURE_SANDBOX_TENANT_ID }}
```

Comment:
Can you sync with whomever is updating us to use managed identity here?

Reply:
@willdavsmith FYI

Reply:
Created this issue: https://github.com/radius-project/samples/issues/1058